.github/workflows/build-windows.yml

```diff
@@ -0,0 +1,201 @@
+name: Build Windows
+
+on:
+  workflow_dispatch:
+  push:
```

Comment:
@devm18426 Here we should include pull requests too. That's what I usually do in the other build workflows 

`on: [push, pull_request]`